docker/Linux-JDK21/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:lunar
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install various dependencies:
+# * ca-certificates is needed by wget
+# * ffmpeg is needed by FfmpegProcessor
+# * wget download stuffs in this dockerfile
+# * libopenjp2-tools is needed by OpenJpegProcessor
+# * All the rest is needed by GrokProcessor
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  ca-certificates \
+  ffmpeg \
+  wget \
+  libopenjp2-tools \
+  liblcms2-dev \
+  libpng-dev \
+  libzstd-dev \
+  libtiff-dev \
+  libjpeg-dev \
+  zlib1g-dev \
+  libwebp-dev \
+  libimage-exiftool-perl \
+  libgrokj2k1 \
+  grokj2k-tools \
+  adduser \
+  openjdk-21-jdk \
+  maven \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install TurboJpegProcessor dependencies
+RUN mkdir -p /opt/libjpeg-turbo/lib
+COPY docker/Linux-JDK11/image_files/libjpeg-turbo/lib64 /opt/libjpeg-turbo/lib
+
+# Install KakaduNativeProcessor dependencies
+COPY dist/deps/Linux-x86-64/lib/* /usr/lib/
+
+# A non-root user is needed for some FilesystemSourceTest tests to work.
+ARG user=cantaloupe
+ARG home=/home/$user
+RUN adduser --home $home $user
+RUN chown -R $user $home
+USER $user
+WORKDIR $home
+
+# Install application dependencies
+COPY ./pom.xml pom.xml
+
+RUN echo "export JAVA_HOME=$(dirname $(dirname $(readlink -f $(which java))))" > ~/.bashrc
+RUN mvn --quiet dependency:resolve
+
+# Copy the code
+COPY --chown=cantaloupe docker/Linux-JDK11/image_files/test.properties test.properties
```

Comment:
same with the test.properties. Should be in a shared space instead inside a specific JDK folder